rollup.config.js

```diff
@@ -0,0 +1,66 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import {terser} from 'rollup-plugin-terser';
+import css from 'rollup-plugin-css-only';
+import execute from 'rollup-plugin-execute';
+
+let plugins = [
+
+    // so Rollup can find externals
+    resolve({extensions: ['.js'], preferBuiltins: true}),
+
+    // so Rollup can convert externals to an ES module
+    commonjs(),
+];
+
+// minify only in production mode
+if (process.env.NODE_ENV === 'production') {
+    plugins.push(
+        // minify
+        terser({
+            ecma: 2018,
+            warnings: true,
+            compress: {
+                drop_console: false,
```

Comment:
Why do you want to maintain `console` statements in a production environment?

Reply:
Fixed in https://github.com/suda/tool-bar/pull/302/commits/a717e6a6854644b9ccb0a791bc6e181fe97bded6